conformance/tests/grpcroute-exact-path-matching.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: GRPCRoute
+metadata:
+  name: exact-matching
```

Comment:
this test is called `path-matching` because the `method` is represented in the path ?